packages/react-components/react-nav-preview/library/src/components/NavDivider/NavDivider.test.tsx

```diff
@@ -1,18 +1,16 @@
-import * as React from 'react';
-import { render } from '@testing-library/react';
 import { isConformant } from '../../testing/isConformant';
 import { NavDivider } from './NavDivider';
 
 describe('NavDivider', () => {
   isConformant({
     Component: NavDivider,
     displayName: 'NavDivider',
-  });
-
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
-  it('renders a default state', () => {
-    const result = render(<NavDivider>Default NavDivider</NavDivider>);
-    expect(result.container).toMatchSnapshot();
+    testOptions: {
+      'has-static-classnames': [
+        {
+          props: { children: 'Test Children' },
```

Comment:
shouldn't the divider render even when there's no children? Another thing is that this is missing the expectedClassNames otherwise it's not checking for anything (I think? 😆 )

Reply:
I think it does render without children. I dunno if we need tests with/without children since that should probably already be covered in regular Divider

Reply:
I was drafting off the divider tests. It seemed the 'wrapper' element doesn't render without children, and the test was looking for wrapper?

Reply:
Ahh yeah, it makes sense for the wrapper, lgtm after the alphabetization of the deps then, I'll approve.